sdc/datatypes/hpat_pandas_getitem_types.py

```diff
@@ -0,0 +1,80 @@
+# *****************************************************************************
+# Copyright (c) 2019, Intel Corporation All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions are met:
+#
+#     Redistributions of source code must retain the above copyright notice,
+#     this list of conditions and the following disclaimer.
+#
+#     Redistributions in binary form must reproduce the above copyright notice,
+#     this list of conditions and the following disclaimer in the documentation
+#     and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+# AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+# OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+# WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR
+# OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+# EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *****************************************************************************
+
+
+import pandas
+
+from numba import types, cgutils
+from numba.extending import (models, overload, register_model, make_attribute_wrapper, intrinsic)
+from numba.datamodel import (register_default, StructModel)
+from numba.typing.templates import signature
+
+
+class SeriesGetitemSelectorType(types.IterableType):
+    def __init__(self, data, selector):
+        self.data = data
+        self.selector = selector
+        super(SeriesGetitemSelectorType, self).__init__('SeriesGetitemSelectorType')
```

Comment:
please try the following code:
`super(SeriesGetitemSelectorType, self).__init__('SeriesGetitemSelectorType({}, {})'.format(data, selector))`

Your issue is the same as @densmirn is fighting with. So, you could ask him about details.

Also, even if it would helps it would berequired to perform some additional tests